recipe/meta.yaml

```
@@ -1,64 +1,65 @@
 {% set name = "blosc" %}
-{% set version = "1.10.6" %}
-{% set sha256 = "55d9d57b85d6eeec010c6c399f2820f96f566dccbc6ddfeefb60501f8e10b548" %}
+{% set version = "1.11.3" %}
 
 package:
   name: python-{{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89ed658eba7814a92e89c44d8c524148d55921595bc133bd1a90f8888a9e088e
+  patches:
+    - numpy-usage.patch
 
 build:
   number: 0
-  # There is no available dependency scikit-build on win32
-  skip: True  # [win32 or s390x]
-  skip: True  # [py<36]
+  skip: True  # [py<39]
   script:
-    - export DISABLE_BLOSC_SSE2=1  # [ppc64le]
-    - export DISABLE_BLOSC_AVX2=1  # [ppc64le]
     - export USE_SYSTEM_BLOSC=1    # [unix]
     - set USE_SYSTEM_BLOSC=1       # [win]
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
     - cmake
```

https://github.com/Blosc/python-blosc/blob/v1.11.3/requirements.txt#L3
```suggestion
    - cmake >=3.14.0
```